scripts/utils/registerInterfaceWithDefaultTheme.js

```diff
@@ -1,3 +1,5 @@
+/* This file should be removed in the next major version update of Rheostat */
```

Comment:
Instead of having this be forked code, can we change this file to be:
```
import registerInterfaceWithDefaultTheme from './lib/utils/registerInterfaceWithDefaultTheme';

export default registerInterfaceWithDefaultTheme;
```
that way we won't have to worry about getting out of sync.